test/runtests.jl

```diff
@@ -210,7 +211,7 @@ end
 
     if Cairo.libcairo_version >= v"1.12.0"
 
-        if ~is_windows()
+        if ~Sys.iswindows()
```

Comment:
@ararslan But that might be a Compat documentation issue 

> is_apple, is_bsd, is_linux, is_unix, and is_windows are now Sys.isapple, Sys.isbsd, Sys.islinux, Sys.isunix, and Sys.iswindows, respectively. These are available in the Compat.Sys submodule.

reads for me like using Compat.Sys and apply Sys.